### PR TITLE
TaskWork-4.5.1

### PR DESCRIPTION
**Pull request by sulfidate**

### CODE REVIEW

File: src/features/filterEventsByCity.test.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        EventListWrapper = AppWrapper.find(Event)
+        expect(EventListWrapper.find('.EventList li')).toHaveLength(0)
+      }
+    )
+
+    and(
+      'the user should receive a list of upcoming events in that city',
+      () => {
+        expect(AppWrapper.find('.event')).toHaveLength(mockData.length)
+      }
```
Comment (stratospherique) [inline src/features/filterEventsByCity.test.js:98]: For these tests, you need to define a variable that contains a filtered version of `mockData`. It contains the events that have 'berlin' in their location string. And the expect instruction would become like this:
`expect(AppWrapper.find('.event')).toHaveLength(filteredEvents.length)`

- The then block serves the same purpose of the `and` block. They could be merged into a single then block.

File: src/features/specifyNumberOfEvents.feature [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    Scenario: number of events input is displayed 
+        Given app opens on main page
+        When number of events is not specified
+        Then number of events input default is displayed
+
+    Scenario: number of events input element changes
+        Given user is on main page
+        When clicks input element
+        Then the number of events input box is displayed
+            And the number of events will specified
```
Comment (stratospherique) [inline src/features/specifyNumberOfEvents.feature:12]: The feature is supposed to test the effect of changing the number of event input on the **number of displayed events**(not on the input itself). That require changes in the associated test file.

**Review by stratospherique — CHANGES_REQUESTED** [review]

File: src/features/filterEventsByCity.test.js (hunk 1 above)
Comment (sulfidate) [reply src/features/filterEventsByCity.test.js:98]: OK Done :)

Comment (sulfidate) [reply src/features/filterEventsByCity.test.js:98]: Done :)

File: src/features/specifyNumberOfEvents.feature (hunk 2 above)
Comment (stratospherique) [reply src/features/specifyNumberOfEvents.feature:12]: The file is still unchanged

Comment (stratospherique) [reply src/features/specifyNumberOfEvents.feature:12]: yes, both files should be refactored.

Comment (sulfidate) [reply src/features/specifyNumberOfEvents.feature]: ok, Done

**Pull request merged**